backend/package.json

```diff
@@ -45,7 +47,10 @@
     "build": "cross-env NODE_ENV=production && webpack",
     "start-prod": "cross-env NODE_PATH=./dist && node dist/main.js",
     "database:populate": "cross-env NODE_PATH=./dist && node dist/populate_database.ts.js",
-    "database:reset": "cross-env NODE_PATH=./dist && node dist/reset.ts.js"
+    "database:reset": "cross-env NODE_PATH=./dist && node dist/reset.ts.js",
+    "prettier:base": "prettier --parser typescript --single-quote --trailing-comma='all' --no-bracket-spacing",
```

Comment:
But is there a need for `prettier:base` and `prettier:check`? Isn't just `prettier:write` enough?